front/app/components/ParticipationCTABars/ParticipationCTAContent/index.tsx

```diff
@@ -110,10 +126,15 @@ export const ParticipationCTAContent = ({
                   ...(isSmallerThanPhone ? { fontWeight: '600' } : {}),
                 }}
               >
-                <FormattedMessage {...userParticipationMessage} />
+                {!hideDefaultParticipationMessage && (
+                  <FormattedMessage {...userParticipationMessage} />
+                )}
+                {hideDefaultParticipationMessage &&
+                  timeLeftPosition === 'left' &&
+                  timeLeftTranslated?.toUpperCase()}
```

Comment:
Was thinking this message should maybe come from the props as well :thinking: Called "participationMessage" or something maybe?